rllib/utils/torch_utils.py

```diff
@@ -114,32 +114,40 @@ def clip_gradients(
     # Clip by value (each gradient individually).
     if grad_clip_by == "value":
         for k, v in gradients_dict.copy().items():
-            gradients_dict[k] = torch.clip(v, -grad_clip, grad_clip)
+            gradients_dict[k] = (
```

Comment:
For torch, some gradients (e.g. from the target networks) are None, so we need to defend for this case here.